released-versions.json

```diff
@@ -1,7 +1,7 @@
 {
     "latestReleaseSnapshot": {
-        "version": "6.1.1-20200124111952+0000",
-        "buildTime": "20200124111952+0000"
+        "version": "6.1.1-20200124000049+0000",
```

Comment:
Why go back here to an older version?

Reply:
It's just caused by the revert, I will use the latest version before merging.